wit_dashboard/wit-dashboard.html

```diff
@@ -7978,7 +7996,16 @@ <h2>Create similarity feature</h2>
         createPdEntryForCurrentValue: function(featureName, data, modelInd) {
           // Extract the class and feature value index from the key of the PD
           // chart data .
-          const key = _.keys(data)[0];
+          const keys = _.keys(data);
+          let key = keys[0];
```

Comment:
does keys[0] correspond to positive class key? (asking because we set key to 1 below if there is labelVocab)

Reply:
added more comments